native/ex_tokenizers/Cargo.toml

```
@@ -14,4 +14,4 @@ anyhow = "1"
 rustler = "0.27.0"
 thiserror = "1"
 regex = "1.5.5"
-tokenizers = { version = "0.11.3", default-features = false, features = []}
+tokenizers = { version = "0.13.3", default-features = false, features = ["default"]}
```

This will include fewer dependencies:

```suggestion
tokenizers = { version = "0.13.3", default-features = false, features = ["onig", "esaxx_fast"]}
```

BUT doing so requires updating the lock, so I pushed it directly to main (sorry :D)